cookbook-react-native/recipes/pickers/dateTimePicker/__test__/components/dateTimePicker/constants.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import {
+  FULL_HOUR,
+  TouchButton,
+  changeDate
+} from '@pickersRecipes/dateTimePicker/constants';
+
+describe('FULL_HOUR', () => {
+  test('return correct value', () => {
+    const expectReturn = 'en_GB';
+    const response = FULL_HOUR(true);
+    expect(response).toBe(expectReturn);
+  });
+});
```

Comment:
i would expect other use cases for each util. In this case what happens when it doesn't receive anything or false

Reply:
writed